packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -447,6 +444,36 @@ export class Pipeline extends PipelineBase {
     return !owner || owner === 'AWS';
   }
 
+  private getOtherStackIfActionIsCrossAccount(action: IAction): Stack | undefined {
```

Comment:
docs (yes I know it's private, but since it's not trivially clear why you need this, don't be shy adding comments)

Reply:
Done.